@here/harp-mapview/lib/MapView.ts

```diff
@@ -2145,6 +2145,13 @@ export class MapView extends EventDispatcher {
         return this.m_tileDataSources;
     }
 
+    /**
+     * Returns the glsl version compatible with the MapView's renderer
+     */
+    get glslVersion(): number {
+        return this.m_renderer.capabilities.isWebGL2 ? 3.0 : 1.0;
+    }
+
```

Comment:
This is a too low-level detail for MapView's interface. We try to move away from using MapView's as a facade for lower level interfaces.
You can access the renderer capabilities doing "mapView.renderer.capabilities", see my other comments.